src/radical/pilot/tmgr/scheduler/backfilling.py

```diff
@@ -11,11 +11,11 @@
 from .base import UMGRSchedulingComponent, ADDED
```

Comment:
Maybe `TMGRSchedulingComponent`?

Reply:
What is that ADDED?

Reply:
A constant defined in `base.py` which signals a pilot state (In the scheduler scope) as `added`.